modules/new_profile/jsx/NewProfileIndex.js

```diff
@@ -103,6 +107,8 @@ class NewProfileIndex extends React.Component {
           });
         } else {
           resp.json().then((message) => {
+            // enable form from resubmission.
```

Comment:
might as well fix the typo here while i  test it manually!

Reply:
@HenriRabalais I fixed it. thanks!